app/angular/src/builders/start-storybook/schema.json

```diff
@@ -67,6 +67,19 @@
       "type": "boolean",
       "description": "Starts Storybook in documentation mode. Learn more about it : https://storybook.js.org/docs/react/writing-docs/build-documentation#preview-storybooks-documentation.",
       "default": false
+    },
+    "compodoc": {
+      "type": "boolean",
+      "description": "Execute compodoc before.",
+      "default": true
```

Comment:
because is added by default with sb init 🤷‍♂️ 
but if the user does not have compodoc it is necessary to manually change to false
I don't know if this is a good situation 🤔

Reply:
@ThibaudAv#1582 does compodoc get installed by default? if so, then we should enable by default. if not, then we should disable by default. open to your opinions, but maybe we will move away from compodoc in the future as requested by some users. but for now i think we do what's consistent with the CLI